front/scripts/test_llm_clients.ts

```diff
@@ -0,0 +1,266 @@
+import { assertNever } from "@dust-tt/sparkle";
+
+import { AnthropicLLM } from "@app/lib/api/llm/clients/anthropic";
+import type { LLMEvent } from "@app/lib/api/llm/types/events";
+import type { ModelIdType, ReasoningEffort } from "@app/types";
+import type { ModelConversationTypeMultiActions } from "@app/types/assistant/generation";
+import {
+  CLAUDE_3_5_HAIKU_20241022_MODEL_ID,
+  CLAUDE_4_5_HAIKU_20251001_MODEL_ID,
+  CLAUDE_4_5_SONNET_20250929_MODEL_ID,
+} from "@app/types/assistant/models/anthropic";
+
+interface TestConfig {
+  provider: "anthropic";
+  modelId: ModelIdType;
+  temperature?: number;
+  reasoningEffort?: ReasoningEffort;
+}
+
+interface TestConversation {
+  name: string;
+  systemPrompt: string;
+  userMessage: string;
```

Comment:
I think here we can have a `ModelConversationTypeMultiActions`. Because later we will test tool output, so we need to have some messages with role assistant.